index.php

```diff
@@ -1,7 +1,10 @@
 <?php
 require 'dbconn.php';
 
-$result = $query->fetchall()
+
+$query->setFetchMode(PDO::FETCH_ASSOC);
```

Comment:
You have already set fetch mode in dbconn.php. I suggest you remove one of them.